config/config_windows.go

```diff
@@ -0,0 +1,42 @@
+// pmm-agent
```

Comment:
Do we support running pmm client in windows OS. As far as I know we don't.

Reply:
So let's delete both files (_unix,_windows), move to config.go and use just unix package?

Reply:
yeah